### PR TITLE
Update to use new core and sdk, fix broken code

### PR DESCRIPTION
**Pull request by MitchTurner**
Context:
> InsufficientMaxFee { max_fee_from_policies: 314900, max_fee_from_gas_price: 314931 }\" }"} error. We need to update SDK to latest

Bumped versions and fixed all the broken pieces.

### CODE REVIEW

File: src/routes.rs [hunk 1]
```diff
@@ -150,7 +150,7 @@ async fn get_coins(
     amount: u64,
 ) -> Result<Vec<Input>, DispenseError> {
     wallet
-        .get_spendable_resources(*base_asset_id, amount)
+        .get_spendable_resources(*base_asset_id, amount, None)
```
Comment (rymnc) [inline src/routes.rs:153]: what is the last value here?

Comment (MitchTurner) [reply src/routes.rs:153]: Honestly, not great docs on it, but if it passes the tests...

**Review by rymnc — COMMENTED** [review]
lgtm, i would suggest to fmt the code, and inspect why `strum_macros` has multiple versions in the `Cargo.lock`

**Review by xgreenx — DISMISSED** [review]

**Comment by MitchTurner** [comment]
> inspect why strum_macros has multiple versions in the Cargo.lock
it looks like we just depend on multiple versions. `fuel-core` and `fuel-asm` depend on different version of `strum` at least.

**Pull request merged**